e2e/package.json

```diff
@@ -10,7 +10,8 @@
   "license": "ISC",
   "devDependencies": {
     "@percy/cypress": "^2.3.1",
-    "cypress": "^4.9.0"
+    "cypress": "^4.9.0",
+    "cypress-pipe": "^1.7.0"
```

Comment:
If we don't need cypress-pipe, can we just revert package.json and package-lock.json?